### DISCUSSION TITLE
Running, debugging tests locally

### DISCUSSION THREAD

**Discussion by BhuvaneshPatil** [promptdesk/promptdesk#48] [General]
I want to set up the tests locally, where I can debug a single test at a time. I am not sure how to do that can you please help me with that?

@justinmacorin

**Comment by justinmacorin** [comment]
Hey @BhuvaneshPatil sure! 

To run the backend tests, try this:
- cd ./backend
- npm run test

To run the front-end test, try this:
- cd ./backend
- npm run dev
- cd ..
- cd ./frontend
- npm run test

Let me know if you have any issues, problems or questions - happy to help!
